libs/feature/teaching/src/lib/json-editor-dialog.tsx

```diff
@@ -127,15 +128,14 @@ export function OpenAsJsonButton({
 	form: UseFormReturn<any>;
 	validationSchema?: ZodSchema;
 }) {
-	const id = useId();
 	const { isJsonEditorOpen, openJsonEditor, onCloseJsonEditor } = useJsonEditor(form);
 
 	return (
-		<button key={id} type="button" className="btn-stroked" onClick={openJsonEditor}>
-			<span>Als JSON bearbeiten</span>
+		<DefaultButton onClick={openJsonEditor}>
+			<span className={"text-gray-600"}>Als JSON bearbeiten</span>
 			{isJsonEditorOpen && (
 				<JsonEditorDialog onClose={onCloseJsonEditor} validationSchema={validationSchema} />
 			)}
-		</button>
+		</DefaultButton>
```

Comment:
```suggestion
		<StrokedButton onClick={openJsonEditor}>
			<span className={"text-gray-600"}>Als JSON bearbeiten</span>
			{isJsonEditorOpen && (
				<JsonEditorDialog onClose={onCloseJsonEditor} validationSchema={validationSchema} />
			)}
		</StrokedButton>
```

ist optisch, mit ausnahme der textgröße gleich. Vermutlich ist der default button nicht notwendig ?